ui/chat/chat.svelte

```diff
@@ -15,6 +16,9 @@
   grooming.onChange(
     state => {
       messages = state.chat;
+      if (state.chat[state.chat.length-1].author !== grooming.username) {
+        audio.play();
```

Comment:
What if that audio URL below becomes invalid? Will `audio.play` throw an error?
What if `audio.play` is called before the file is loaded?

Reply:
Sound will not play and we can see an error in the console, that is all, the app will not be crashed. I added check that audio is available for playing.